File: Linguagem C++/Estrutura Repetitiva/Tabuada.cpp
#include <iostream>

using namespace std;

int main()
{
    int n,soma;

    cout << "Deseja a tabuada para qual valor? ";
    cin >> n;

    for (int i=1 ; i<=10 ; i++){
        soma = n*i;
        cout << n << " x " << i << " = " << soma << endl;
    };


    return 0;
}

File: Linguagem Java/Exercicios/Tabuada.java
import java.util.Locale;
import java.util.Scanner;

public class Tabuada {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n,soma;
		
		soma = 0;
		System.out.print("Deseja a tabuada para qual valor? ");
        n = sc.nextInt();
        
        for (int i=1;i<=10;i++) {
        	soma = n*i;
        	System.out.println(n+" x "+i+" = "+soma);
        };
        
        sc.close();
	}

}
